lib/util.py

```diff
@@ -503,6 +503,8 @@ def time_difference(distance_in_time, include_seconds):
                         {'tx': 'tx', 'addr': 'address'}),
     'explorer.feathercoin.com': ('http://explorer.feathercoin.com/',
                         {'tx': 'tx', 'addr': 'address'}),
+    'chainz.cryptoid.info': ('https://chainz.cryptoid.info/ftc/',
+                        {'tx': 'tx', 'addr': 'address'}),
```

Comment:
I've tested this patch and unfortunately it still doesn't work properly. If you take a look at the following two URLs
[https://chainz.cryptoid.info/ftc/tx.dws?cc633c7007283951939995816e2fd58168e82d8afafb0b84beba573287a56f98](https://chainz.cryptoid.info/ftc/tx.dws?cc633c7007283951939995816e2fd58168e82d8afafb0b84beba573287a56f98)
[https://chainz.cryptoid.info/ftc/address.dws?6myoj8eimVsaneikcZ5WRHxgyrxsBbyNBy](https://chainz.cryptoid.info/ftc/address.dws?6myoj8eimVsaneikcZ5WRHxgyrxsBbyNBy)
you can see that `.dws?` also needs to be appended before the transaction hash or address, respectively. Therefore, this line should be
```py
                       {'tx': 'tx.dws?', 'addr': 'address.dws?'}),
```

Reply:
oh... didn't pay enough attention to that detail, should have looked at the URLs first.
Thanks anyway, I added those two aswell now.